Lombiq.DataTables/Tests/Lombiq.DataTables.Tests/UnitTests/Services/ExportTests.cs

```diff
@@ -30,6 +33,14 @@ public class ExportTests
         int length,
         int orderColumnIndex)
     {
+        // ClosedXML looks at the CurrentCulture to initialize the workbook's culture. They also to set it like this
+        // in their own unit tests. See:
+#pragma warning disable S103 // Split this long line.
+        // https://github.com/ClosedXML/ClosedXML/blob/c2d408b127844ea3d4a5f6b060c548c953b6bcf3/ClosedXML_Tests/Excel/CalcEngine/LookupTests.cs#L16-L17
+#pragma warning restore S103 // Split this long line.
+        // As this is an async method, it runs in its own thread so this has no effect on other tests.
```

Comment:
Rewrapping + grammar improvements:
```suggestion
        // ClosedXML looks at the CurrentCulture to initialize the workbook's culture. They also to set it like this in
        // their own unit tests. See:
#pragma warning disable S103 // Split this long line.
        // https://github.com/ClosedXML/ClosedXML/blob/c2d408b127844ea3d4a5f6b060c548c953b6bcf3/ClosedXML_Tests/Excel/CalcEngine/LookupTests.cs#L16-L17
#pragma warning restore S103 // Split this long line.
        // Since this is an async method, it runs in its own thread; so this has no effect on other tests.
```